src/wolff_client.erl

```diff
@@ -37,10 +37,12 @@
 
 -type config() :: map().
 -type topic() :: kpro:topic().
+-type alias_and_topic() :: wolff_producers:alias_and_topic().
```

Comment:
is it possible to keep alias at `wolff_producers` level ?
i.e. share the leader connections between different producers.

Reply:
I think not without further changes, because of this:

https://github.com/kafka4beam/wolff/blob/a5cbf24d76e83c858ec3892fe70866e243598890/src/wolff_producers.erl#L137-L141

In order for this to be idempotent, we'll need to do some kind of reference counting by alias.  I'll see if I managed to do it.